boards/arm/acn52832/doc/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _acn52832:
+
+acn52832
+########
+
+Overview
+********
+
+The acn52832 is a module in a small form factor which features the Nordic Semiconductor
```

Comment:
This documentation is incredibly light, can you look at documentation for other boards and add more to it?

Reply:
I added some more documentation.